src/mudlet-lua/lua/Other.lua

```diff
@@ -636,3 +636,29 @@ function shms(seconds, bool)
 		return hh, mm, ss
 	end
 end
+
+-- returns true if your Mudlet is older than the given version
+-- for example, it'll return true if you're on 2.1 and you do mudletOlderThan("3.1")
+-- it'll return false if you're on 4.0 and you do mudletOlderThan("4.0.0")
+-- credit to https://stackoverflow.com/a/16187766/72944 for the algorithm
+function mudletOlderThan(version)
+  local tonumber = tonumber
+	
+	-- strip trailing zeros from input version
+  local input = rex.gsub(version, [[(\.0+)+$]], ''):split('%.')
+	local mudlets = {getMudletVersion("table")}
```

Comment:
Why not use `local major, minor, patch = getMudletVersion()` to get just the __numbers__ you want...?